crates/rpc/src/handlers/validator.rs

```diff
@@ -0,0 +1,97 @@
+use ream_consensus::validator::Validator;
+use ream_storage::db::ReamDB;
+use serde::{Deserialize, Serialize};
+use warp::{
+    http::status::StatusCode,
+    reject::Rejection,
+    reply::{Reply, with_status},
+};
+
+use super::{BeaconResponse, state::get_state_from_id};
+use crate::types::{
+    errors::ApiError,
+    id::{ID, ValidatorID},
+};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct ValidatorData {
+    index: usize,
+    balance: u64,
```

Comment:
```suggestion
    #[serde(with = "serde_utils::quoted_usize")]
    index: usize,
    #[serde(with = "serde_utils::quoted_u64")]
    balance: u64,
```

These are being serialized wrong